app/views/confirm_submission.scala.html

```diff
@@ -116,14 +116,19 @@ <h3 class="bold-small form-group">@messages("confirm_submission.additionalInfo")
       <div class="form-group">
         <h3 class="bold-large">@messages("confirm_submission.returnTotal")@displayMoney(viewModel.returnDetail.box5)</h3>
         <p>@messages("confirm_submission.returnDueDate", displayDate(viewModel.returnDetail.due))</p>
-        <a href=@controllers.routes.SubmitFormController.show(viewModel.periodKey)>@messages("confirm_submission.changeDetail")</a>
+        <a
+          onClick="sendGAEvent('submit-vat', 'change', 'nine-box-change' @user.isAgent)"
+          href=@controllers.routes.SubmitFormController.show(viewModel.periodKey)>
+          @messages("confirm_submission.changeDetail")
+        </a>
```

Comment:
Hi buddy, could you add this event to the confluence page please: https://confluence.tools.tax.service.gov.uk/pages/viewpage.action?pageId=150933079

Reply:
I forgot to publish the changes. Just update it for you